OpenRA.Mods.Common/Widgets/Logic/Ingame/GameTimerLogic.cs

```diff
@@ -32,12 +44,12 @@ public GameTimerLogic(Widget widget, OrderManager orderManager, World world)
 			Func<string> statusText = () =>
 			{
 				if (world.Paused || world.ReplayTimestep == 0)
-					return "Paused";
+					return modData.Translation.GetString(Paused);
```

Comment:
Pull this and `MaxSpeed` outside the `Func<string> statusText` and use a `CachedTransform` for `Speed`.